### PR TITLE
Added kubernetes deployment for telegrambot

### PR DESCRIPTION
**Pull request by AliAyub007**
Fixes issue #24 

Changes: 
Added kubernetes deployment for SUSI telegram bot.

### CODE REVIEW

**Comment by AliAyub007** [comment]
@niranjan94 @chiragw15 @uday96 Kindly review this PR. Thanks

File: kubernetes/yamls/generator/configmap.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+metadata:
+  name: susi-telegrambot
+  namespace: web
+data:
+  token: "XXXXXXXXXXXXXXXXXXXXXXXXXXXX"
```
Comment (uday96) [inline kubernetes/yamls/generator/configmap.yml]: Is it a dummy token?

Comment (AliAyub007) [inline kubernetes/yamls/generator/configmap.yml]: @uday96 Yes it is dummy token.

File: kubernetes/images/generator/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,22 @@
+FROM node:boron
+
+ARG COMMIT_HASH
+ARG BRANCH
+ARG REPOSITORY
+
+ENV COMMIT_HASH ${COMMIT_HASH:-null}
+ENV BRANCH ${BRANCH:-development}
+ENV REPOSITORY ${REPOSITORY:-https://github.com/AliAyub007/susi_telegrambot.git}
```
Comment (chiragw15) [inline kubernetes/images/generator/Dockerfile]: Shouldn't it be `fossasia//susi_telegrambot.git` ?

File: kubernetes/images/generator/setup.sh [hunk 3]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+git clone ${REPOSITORY} susi_telegrambot
+cd susi_telegrambot
+git checkout ${BRANCH}
+
+if [ -v COMMIT_HASH ]; then
+    git reset --hard ${COMMIT_HASH}
+fi
+
```
Comment (chiragw15) [inline kubernetes/images/generator/setup.sh]: Here delete .git files to reduce size of docker image. Use `rm -rf .git`

File: kubernetes/travis/deploy.sh [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+sudo rm -f /usr/bin/gsutil
+sudo rm -f /usr/bin/gcloud
+rm -rf node_modules
+
+curl https://sdk.cloud.google.com | bash;
+source ~/.bashrc
+gcloud components install kubectl
+
+gcloud config set compute/zone us-central1-b
+openssl aes-256-cbc -K $encrypted_324c873b2450_key -iv $encrypted_324c873b2450_iv -in ./kubernetes/travis/susi-telegrambot-e467bca1e540.json.enc -out susi-telegrambot-e467bca1e540.json -d
```
Comment (chiragw15) [inline kubernetes/travis/deploy.sh]: Make sure you have added these keys in travis repo settings of main repo and not in your fork's travis settings. You will need admin rights for that. We faced this problem while deploying susi server.

File: kubernetes/images/generator/Dockerfile (hunk 2 above)
Comment (AliAyub007) [inline kubernetes/images/generator/Dockerfile]: Yes, I had setup for my that's why I will change it.

File: kubernetes/travis/deploy.sh (hunk 4 above)
Comment (AliAyub007) [inline kubernetes/travis/deploy.sh]: Yes I will shift them.

File: kubernetes/travis/deploy.sh [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+mkdir -p lib
+gcloud auth activate-service-account --key-file susi-telegrambot-e467bca1e540.json
+export GOOGLE_APPLICATION_CREDENTIALS=$(pwd)/susi-telegrambot-e467bca1e540.json
+gcloud config set project susi-telegrambot
+gcloud container clusters get-credentials bots
+cd kubernetes/images/generator
+docker build --build-arg COMMIT_HASH=$TRAVIS_COMMIT --build-arg BRANCH=$DEPLOY_BRANCH --build-arg REPOSITORY=$REPOSITORY --no-cache -t aliayubkhan/susi_telegrambot:$TRAVIS_COMMIT .
+docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD"
+docker tag aliayubkhan/susi_telegrambot:$TRAVIS_COMMIT aliayubkhan/susi_telegrambot:latest
```
Comment (chiragw15) [inline kubernetes/travis/deploy.sh]: @AliAyub007 here, change this to fossasia docker account and push docker image to it. I am creating a new repository in docker fossasia organization and messaging you the credentials of susi docker account. Store those credentials as travis environment variables. Also, use tag as `latest-$DEPLOY_BRANCH`

**Pull request merged**